src/Makefile.in

```diff
@@ -840,7 +840,7 @@ zsh-completion:
 shell-completion: bash-completion zsh-completion
 
 kernel-cache-clean:
-	$(RM) -fr ~/.nv/ComputeCache ../run/opencl/*.bin
+	$(RM) -fr ~/.nv/ComputeCache ../run/opencl/*.bin ../run/opencl/*.config
```

Comment:
Wouldn't Makefle.legacy need a similar change?

Reply:
True, thanks - I'll add it